testData/solutions/win32.json

```diff
@@ -1426,8 +1786,9 @@
                     "highContrastEnabled": "http://registry\\.gpii\\.net/common/highContrastEnabled",
                     "highContrastTheme": "http://registry\\.gpii\\.net/common/highContrastTheme",
                     "invertColours": "http://registry\\.gpii\\.net/common/invertColours"
-                }
-           }
+                },
+                "inverseCapabilitiesTransformations": {}
```

Comment:
now added this to the documentation on the solution registry format (present in the /documentation folder of this github repo)